go.mod

```diff
@@ -1,150 +1,157 @@
 module github.com/opendatahub-io/odh-model-controller
 
-go 1.22.7
-
-toolchain go1.22.9
+go 1.23.6
```

Comment:
I think @VedantMahabaleshwarkar opened a PR for it, may we remove it from here?
Or close his PR?

Reply:
Rebased